plugins/rikki/loungestatistics/components/SlothStatistics.php

```diff
@@ -24,19 +24,49 @@ public function componentDetails()
     public $sloth = null;
     public $heroes = null;
     public $maps = null;
-    public $allseasons = null;
+    public $participatedSeasons = null;
+    public $selectedSeason = null;
 
     public function init()
     {
         $this->addJs('assets/js/datatables.min.js');
         $this->addJs('assets/js/loungestatistics.js');
         $this->addCss('assets/css/datatables.min.css');
+
         $this->sloth = Sloth::where('id', $this->property('sloth_id'))
                         ->with('gameParticipations', 'gameParticipations.team', 'gameParticipations.game', 'gameParticipations.game.map', 'gameParticipations.game.match', 'gameParticipations.hero', 'gameParticipations.game.teamOneFirstBan', 'gameParticipations.game.teamOneSecondBan', 'gameParticipations.game.teamTwoFirstBan', 'gameParticipations.game.teamTwoSecondBan')
                         ->first();
-        $this->allseasons = Season::where('reg_open', false)->get()->sortByDesc('created_at');
 
-        $this->calculateStats($this->allseasons->first());
```

Comment:
did this line ever show anything?  It does not render to the partial like `onSeasonChange` below

Reply:
I was not clicking the "toggle" buttons, mb